nimare/meta/cbma/base.py

```diff
@@ -94,6 +95,9 @@ def _fit(self, dataset):
         else:
             self._compute_null_reduced_empirical(ma_values, n_iters=self.n_iters)
 
+        if self.kernel_transformer.low_memory:
+            os.remove(ma_values.filename)
```

Comment:
@jdkent WDYT of this for cleanup?

Reply:
It turns out that was a bad idea, and the kernel transformers can handle the memmap deletion internally.